package.json

```diff
@@ -38,58 +38,58 @@
     }
   ],
   "devDependencies": {
-    "@babel/cli": "^7.10.1",
-    "@babel/core": "^7.10.2",
-    "@babel/preset-env": "^7.10.2",
-    "babel-loader": "^8.1.0",
+    "@babel/cli": "^7.12.10",
+    "@babel/core": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
+    "babel-loader": "^8.2.2",
     "babelify": "^10.0.0",
-    "browserify": "^16.5.1",
+    "browserify": "^17.0.0",
     "concat-stream": "^2.0.0",
     "envify": "^4.1.0",
     "envify-loader": "^0.1.0",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-plugin-prettier": "^3.1.3",
+    "eslint": "^7.18.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-plugin-prettier": "^3.3.1",
     "expect.js": "^0.3.1",
     "jquery": "^3.5.1",
-    "jsdoc": "^3.6.4",
-    "karma": "^4.4.1",
-    "karma-browserify": "^7.0.0",
+    "jsdoc": "^3.6.6",
+    "karma": "^6.0.2",
+    "karma-browserify": "^8.0.0",
     "karma-chrome-launcher": "^3.1.0",
     "karma-eslint": "^2.2.0",
     "karma-express-http-server": "0.0.1",
-    "karma-firefox-launcher": "^1.3.0",
-    "karma-mocha": "^1.3.0",
+    "karma-firefox-launcher": "^2.1.0",
+    "karma-mocha": "^2.0.1",
     "memory-fs": "^0.5.0",
-    "mocha": "^7.2.0",
-    "prettier": "^1.19.1",
+    "mocha": "^8.2.1",
+    "prettier": "^2.2.1",
     "serve-static": "^1.14.1",
-    "sinon": "^9.0.2",
+    "sinon": "^9.2.4",
     "transform-loader": "^0.2.4",
-    "uglify-js": "^3.9.4",
+    "uglify-js": "^3.12.5",
     "watchify": "^3.11.1",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
+    "webpack": "^5.18.0",
+    "webpack-cli": "^4.4.0"
   },
   "dependencies": {
     "@babel/polyfill": "^7.10.1",
     "buffer-from": "^1.1.1",
-    "camelcase": "^5.3.1",
+    "camelcase": "^6.2.0",
     "clone": "^2.1.2",
-    "core-js": "^3.6.5",
+    "core-js": "^3.8.3",
     "defaults": "^1.0.3",
     "get-user-media-promise": "^1.1.4",
-    "ibm-watson": "^5.6.0",
+    "ibm-watson": "^6.0.2",
```

Comment:
Should we make this a feature release since you are doing a major release of the ibm-watson sdk?

Reply:
There is a separate issue [here](https://github.ibm.com/Watson/developer-experience/issues/9005) asking for a proper 1.0 release of this sdk; however, upgrading to `ibm-watson 6.0.2` doesn't actually release any new features to this sdk so I don't think the upgrade itself would justify a feature release

Reply:
didn't we drop support for Node <10 in ibm-watson 6.x?

Reply:
We did and I can update the package.json to reflect that but I don't see what new features that would give the user in terms of using the sdk and its methods

Reply:
Nevermind Angelo